source/git-api.js

```diff
@@ -106,12 +108,21 @@ exports.registerApi = (env) => {
       return false;  // ignore files that are in .gitignore
     } else if (filename.endsWith(".lock")) {
       return false;
-    } else if (filename.indexOf(path.join(".git", "refs")) > -1) {
-      return true;   // trigger for all changes under refs
-    } else if (filename == path.join(".git", "HEAD")) {
-      return true;   // Explicitly return true for ".git/HEAD" for branch changes
+    } else if (filename.indexOf(path.join("", "refs")) > -1) {
```

Comment:
Let's remove empty string as it doesn't needs to be there.

Reply:
Done